wiki/Storyboard/Scripting/General_Rules/en.md

```diff
@@ -2,103 +2,109 @@
+*For objects in [osu!](/wiki/Game_mode/osu!) and [Beatmapping](/wiki/Beatmapping), see: [Hit Objects](/wiki/Gameplay/Hit_object)*
+
+A [Storyboard object](/wiki/Storyboard/Scripting/Objects) is an instance of a sprite or an animation in a storyboard. Storyboards can also have sound, see the [Audio](/wiki/Storyboard/Scripting/Audio) guide for more details.
+
+The accepted formats for objects are PNG and JPEG. JPEG is lossy, which means it is smaller in file size, but does not precisely save each pixel. It also does not support transparency. Therefore, it is good for backgrounds and for square or photo-realistic images. PNG is lossless, which means it retains pixel-by-pixel information, but is of a larger file size than JPEG. It supports transparency, so it is usually the best for foreground objects / text.
+
+Animations are done in-engine, so the PNG layer system or animation features should not be used. Instead, save each frame as a separate file and name the files with a decimal number before the extension (e.g., "sample0.png", "sample1.png" for a 2-frame animation "sample.png").
 
 ### Screen size
 
-![Editor screen size. Green is screen size and Red is playarea](img/SBS_SS.jpg "Editor screen size. Green is screen size and Red is playarea")
+![Editor screen size. Green is screen size and Red is play area](img/SBS_SS.jpg "Editor screen size. Green is screen size and Red is play area")
+
+The editor screen is 640 x 480 pixels and the general play area is 510 x 385 pixels.
+
+Coordinates are specified with positive values for `X` going to the **right**, positive values for `Y` going **down**, and the origin (0,0) being placed at the upper-left corner of the screen. This is different from traditional [Euclidean coordinate systems](https://en.wikipedia.org/wiki/Euclidean_space), but is the same as most computer-graphics systems. It is possible to specify coordinates to be outside of these boundaries (e.g., for a sprite to come in from off-screen).
 
-- The screen is 640 pixels wide by 480 pixels tall `(640x480)`.
-  - General playarea is 510 pixels wide by 385 pixels tall `(510x385)`.
-- Coordinates are specified with X going to the **right**, Y going **down**, and the origin (0,0) at the upper-left corner of the screen. This is different from traditional Euclidean coordinate systems, but is the same as most computer-graphics systems.
-- You are allowed to specify coordinates outside of these boundaries (e.g., for a sprite to come in from off-screen).
-- The coordinates of your cursor will be displayed in the Design tab in the [Beatmap Editor](/wiki/Client/Beatmap_editor).
+The cursor's coordinates will be displayed in the [Design tab](/wiki/Client/Beatmap_editor/Design) of the [Beatmap Editor](/wiki/Client/Beatmap_editor).
```

Comment:
Not sure about this one here either. The coordinates are visible in both the `compose` and the `design` tab, as visible in the picture above.